GirafRest/Controllers/UserController.cs

```diff
@@ -475,6 +476,9 @@ public async Task<ActionResult> GetCitizens(string id)
                 citizens.Add(new DisplayNameDTO { UserId = girafUser.Id, DisplayName = girafUser.DisplayName });
             }
 
+            //sort function for users in citizens
```

Comment:
Denne kommentar beskriver hvad der sker men ikke hvorfor.
Jeg tror det bliver gjort fordi Weekplanneren forventer alfabetisk rækkefølge, men jeg er ikke sikker.
Hvis det er tilfældet så er kommentaren "// Sorts the users by name because the order is expected by the Weekplanner" mere fortællende.